[core]/es_extended/config.lua

```diff
@@ -63,6 +63,8 @@ Config.RemoveHudCommonents = {
 	[22] = false, --HUD_WEAPONS
 }
 
+Config.EnableGetSharedObjectEvent = false -- This is not recommended, Enables old getSharedObject event.
```

Comment:
```suggestion
Config.EnableGetSharedObjectEvent = false -- Warning: Enabling This WILL decrease your servers performance, use this at your own precaution.
```